README.md

```diff
@@ -1,3 +1,4 @@
+![](https://github.com/lemmaa/jerryscript/blob/master/LOGO.png)
 # JerryScript: JavaScript engine for Internet of Things
```

Comment:
Shouldn't this url point into the Samsung/jerryscript main repo? (Once this PR gets merged, it will become valid.)